addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -68,8 +69,14 @@ def _l10n_eg_eta_connect_to_server(self, request_data, request_url, method, is_a
                 'error': str(ex),
                 'blocking_level': 'warning'
             }
-        if not request_response.ok:
+        try:
             response_data = request_response.json()
+        except JSONDecodeError as ex:
+            return {
+                'error': isinstance(request_response, str) and request_response or str(ex),
+                'blocking_level': 'warning'
```

Comment:
you should justify in commit description why it should be `warning` value here.

Reply:
Thank you for pointing it out sir. I have done the changes and also specified the reason behind it.